api-report/container-runtime.api.md

```diff
@@ -628,6 +626,14 @@ export type OpActionEventListener = (op: ISequencedDocumentMessage) => void;
 // @public (undocumented)
 export type OpActionEventName = MessageType.Summarize | MessageType.SummaryAck | MessageType.SummaryNack | "default";
 
+// @public (undocumented)
+export enum OrderSequentiallyFailureMode {
```

Comment:
i don't think we should expose this until the dds layers are ready

Reply:
you can use a config for testing

Reply:
Are you thinking it should be exposed when all DDSes support? Or just after one DDS?

Reply:
i don't have a clear answer there. it would be nice if all our dds supported it, but we could ship the feature before all dds support it if there is a reason to. right now just seems too soon, as no dds support it.